packages/gatsby/src/utils/webpack.config.js

```diff
@@ -614,7 +615,6 @@ module.exports = async (
       `@reach/router/lib/history`,
       `@reach/router`,
       `common-tags`,
-      /^core-js\//,
```

Comment:
why don't we want this external anymore? or does the resolver do this somehow?

Reply:
I saw this break sometimes with gatsby-dev, never with a regular `yarn install`, so more or less just a precaution.

Reply:
So should we bring it back then before we ship? My understanding of this array (based on the comment above) is that we force these libs into a common bundle to not bloat page bundles. Doesn't removing this line put core-js in each bundle?

Reply:
Before we weren't merging core-js into SSR bundles to reduce size. I'll try to add it back and see what tests say.